### PR TITLE
Feat/doc

### PR DESCRIPTION
**Pull request by charnysh-dev**

### CODE REVIEW

File: src/utils/shared/redirects.test.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  describe('fromSubdomains', () => {
+    // Remove www (when already HTTPS)
+    itRedirects('https://www.cml.dev/foo', 'https://cml.dev/foo')
+  })
+
+  describe('toDiscord', () => {
+    itRedirects('/help', 'https://discordapp.com/invite/dvwXA2N', 303)
+
+    itRedirects('/chat', 'https://discordapp.com/invite/dvwXA2N', 303)
+  })
```
Comment (rogermparent) [inline src/utils/shared/redirects.test.js:108]: Looks like some of these redirect tests are outdated, seems like it'll just be a matter of removing any of them that aren't in `redirects-list.json`.

**Review by julieg18 — COMMENTED** [review]
Nice job on this! I don't see any major issues.

**Comment by casperdcl** [comment]
Fonts & inverted banner colours seems at odds with https://dvc.org/doc but in terms of engine features looks about right to me.

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,81 @@
+# Javascript Node CircleCI 2.0 configuration file
```
[the remark this answers is not in the document]
Comment (shcheklein) [reply .circleci/config.yml:1]: we are keeping this as close to dvc.org as possible for now

File: .pre-commit-hooks.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+- id: dead-url
```
[the remark this answers is not in the document]
Comment (shcheklein) [reply .pre-commit-hooks.yaml:1]: same. Comes from dvc.org - if we want to solve it, let's open a ticket and discuss there . Husky reads this file?

File: .prettierignore [hunk 4]
```diff
@@ -1,5 +1,4 @@
-.cache
-package.json
-package-lock.json
-public
-storybook-static*
+.cache/
+.github
```
[the remark this answers is not in the document]
Comment (shcheklein) [reply .prettierignore:2]: 👍

File: LICENSE [hunk 5]
```diff
@@ -1,22 +1,201 @@
-The MIT License (MIT)
+                                 Apache License
```
[the remark this answers is not in the document]
Comment (shcheklein) [reply LICENSE:1]: yes

File: .prettierrc [hunk 6]
```diff
@@ -1,4 +1,10 @@
 {
+  "semi": false,
   "arrowParens": "avoid",
-  "semi": false
+  "singleQuote": true,
+  "trailingComma": "none",
+  "printWidth": 80,
+  "tabWidth": 2,
+  "useTabs": false,
+  "proseWrap": "always"
```
[the remark this answers is not in the document]
Comment (shcheklein) [reply .prettierrc:9]: this PR completely gets rid of the previous website and takes dvc.org as a base. It makes sense to keep the relevant logic/configs from it I think.

File: .restyled.yaml [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+        '--print-width=80',
+        '--tab-width=2',
+        '--use-tabs=false',
+        '--prose-wrap=always',
+      ]
+    include:
+      - './*.{js,md}'
+      - 'pages/**/*.js'
+      - 'content/**/*.md'
+      - 'src/**/*.js'
```
[the remark this answers is not in the document]
Comment (jorgeorpinel) [reply .restyled.yaml:19]: Yes, it's a white list. For example when you run the dev version there's a .cache/ folder with lots of .js files; there can be .vscode/extensions.json in your repo; etc.

**Pull request merged**